ts/IterableInboxMessageDisplay.tsx

```diff
@@ -101,7 +102,11 @@ const IterableInboxMessageDisplay = ({
    return(
       <View style={updatedMessageDisplayContainer}>
          <View style={returnButtonContainer}>
-            <TouchableWithoutFeedback onPress={() => returnToInbox()}>
+            <TouchableWithoutFeedback 
+               onPress={() => {
+                  returnToInbox()
+                  Iterable.trackInAppClose(rowViewModel.inAppMessage, IterableInAppLocation.inbox, IterableInAppCloseSource.back, 'iterable://dismiss')
+               }}>
```

Comment:
Same to here, we should allow `clickedUrl` to be `null`.

Reply:
I will make the update.

Reply:
I added undefined for the clickedUrl field.